2019-12-19 06:24:06,292 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 43242798ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-19 08:15:03,615 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 6656338ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-19 10:49:57,755 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 9294126ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-19 12:18:03,838 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 825226ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-19 12:48:39,463 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 1713569ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-19 13:24:18,133 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 2118627ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-19 14:07:19,578 ERROR 44029 nodejs.ClusterClientNoResponseError: client no response in 440890ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/pengzefan/learn/shopping-system/be/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:536:17)
    at processTimers (internal/timers.js:480:7)
name: "ClusterClientNoResponseError"
pid: 44029
hostname: C02Y81Q2JG5L

2019-12-19 17:27:40,641 ERROR 65300 [-/127.0.0.1/-/24ms GET /recommend-goods] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'isDelete' in 'where clause'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: SELECT * FROM `goods` WHERE `isDelete` = 0 ORDER BY `id` LIMIT 0, 6
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'isDelete' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `goods` WHERE `isDelete` = 0 ORDER BY `id` LIMIT 0, 6"
name: "ER_BAD_FIELD_ERRORError"
pid: 65300
hostname: C02Y81Q2JG5L

2019-12-19 17:27:54,781 ERROR 65300 [-/127.0.0.1/-/2ms GET /recommend-goods] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'isDelete' in 'where clause'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: SELECT * FROM `goods` WHERE `isDelete` = 0 ORDER BY `id` LIMIT 0, 6
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'isDelete' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `goods` WHERE `isDelete` = 0 ORDER BY `id` LIMIT 0, 6"
name: "ER_BAD_FIELD_ERRORError"
pid: 65300
hostname: C02Y81Q2JG5L

2019-12-19 17:31:04,632 ERROR 65369 [-/127.0.0.1/-/7ms GET /recommend-goods] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'random' in 'order clause'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: SELECT * FROM `goods` WHERE `isDeleted` = 0 ORDER BY `random` LIMIT 0, 6
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'random' in 'order clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `goods` WHERE `isDeleted` = 0 ORDER BY `random` LIMIT 0, 6"
name: "ER_BAD_FIELD_ERRORError"
pid: 65369
hostname: C02Y81Q2JG5L

2019-12-19 17:34:04,542 ERROR 65369 [-/127.0.0.1/-/2ms GET /recommend-goods] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'random' in 'order clause'
    at Query.Sequence._packetToError (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:219:5)
    at addChunk (_stream_readable.js:332:12)
    --------------------
    at Pool.query (/Users/pengzefan/learn/shopping-system/be/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/pengzefan/learn/shopping-system/be/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:65:19)
    at /Users/pengzefan/learn/shopping-system/be/node_modules/co/index.js:54:5
    sql: SELECT * FROM `goods` WHERE `isDeleted` = 0 ORDER BY `random` LIMIT 0, 6
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'random' in 'order clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `goods` WHERE `isDeleted` = 0 ORDER BY `random` LIMIT 0, 6"
name: "ER_BAD_FIELD_ERRORError"
pid: 65369
hostname: C02Y81Q2JG5L

